src/Skautis.php

```diff
@@ -28,7 +30,7 @@ class Skautis
     private $user;
 
     /**
-     * @var SkautisQuery[]
+     * @var SkautisQuery[]|null
```

Comment:
Nebylo by lepší mít jako výchozí hodnotu `[]` a nepovolovat null?

Reply:
Null ma specialni vyznam - nelogovat. V budoucnu muzeme pridat extra flag a tady dat vychozi hodnotu.

Reply:
Chápu. S tím komentářem už je to super. :+1: